### ISSUE TITLE
Allow user to know imagery loading state

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#7451]
A black background might represent
1. imagery loading... please wait
2. imagery not available for given imagery choice and zoom level
3. user has chosen "None" imagery
4. some other condition I am not aware of

At least conditions 1 and 2 should be distinguishable via the use of for 1) "Loading" and/or for 2) "Zoom out to see imagery" etc. watermark backgrounds.

**Comment by quincylvania** [comment]
@jidanni Is this actually a usability problem for you? You can just pan and zoom around to get a sense of what the imagery is doing. To my knowledge, mappers have managed fine thus far with the existing behavior.

**Comment by jidanni** [comment]
Ah, but that assumes high speed Internet connections and perfect imagery servers.
Anything else and the user is wondering if he should just wait, or if waiting will be fruitless, for a given black area.

**Comment by jidanni** [comment]
Yes we don't want to waste even more cycles by putting spinning things there, but maybe there could be a default background that eventually gets replaced by imagery.

**Comment by kallejre** [comment]
As someone experimenting with custom layers, 4th category can include incorrect layer settings (incorrect url, servers replies with custom error, zoom level not available). That's unlikely to happen for end user, but i agree, that empty black tile should include something, for example dark grey error code/message, except for None-background.

Temporary workaround is to use browser console's network traffic tab. And look server responses there

**Comment by jidanni** [comment]
See also #8444.
